.changeset/fair-poets-watch.md

```diff
@@ -0,0 +1,11 @@
+---
+'@commercetools-frontend/mc-dev-authentication': minor
+'@commercetools-frontend/mc-html-template': minor
+'@commercetools-frontend/mc-scripts': minor
+---
+
+Enable opt-in support for experimental bundler (using [Vite.js](https://vitejs.dev/)). To enable it, set the environment variable `ENABLE_EXPERIMENTAL_BUNDLER="true"` in your dotenv file.
```

Comment:
```suggestion
Enable opt-in support for experimental bundler (using [Vite.js](https://vitejs.dev/)). To enable it, set the environment variable `ENABLE_EXPERIMENTAL_VITE_BUNDLING=true"` in your dotenv file.
```

Just as we might have the flag for a bit to make it really clear what it turns on.